src/ip/QosConfig.h

```diff
@@ -61,6 +61,15 @@ namespace Ip
 namespace Qos
 {
 
+/**
+ * Specifies connection type (from/to Squid).
+ * Used for getting CONNMARK by getNfmarkFromConnection()
+ */
+enum ConnType {
+    Incoming,
+    Outgoing
```

Comment:
Avoid the terms "incoming" and "outgoing" because a packet sent by Squid to the client is treated as an "outgoing" packet in some contexts. It would be awkward to deal with outgoing packets on incoming connections :-). 

It is a good idea to use a common prefix for enum values, especially when they are placed in a relatively large (and mostly unrelated!) context like Qos. There is a good change this enum will be moved to Connection.h (or similar) later.

See another comment for specific suggestions (but you may be able to find better prefixes/names).